README.md

```diff
@@ -53,9 +53,30 @@ and also provide the foundation that LunaSec builds on to offer you additional s
 
 You can read more [here](https://www.lunasec.io/docs/pages/overview/features/) about what features LunaSec provides.
 
+## Live Demo
```

Comment:
We might want to say right here that

"If you just want to see lunasek running in a browser, click here to see the hosted demo.  If you want to run it locally..."

Reply:
I will when the Live Demo actually exists ;)